docs/developer-guide/ksqldb-clients/java-client.md

```diff
@@ -103,6 +103,12 @@ Start by creating a `pom.xml` for your Java application:
 </project>
 ```
 
+!!! note
+      If you’re using ksqlDB for Confluent Platform (CP), use the CP-specific modules,
+      available at `http://packages.confluent.io/maven/`, by replacing the repositories
```

Comment:
```suggestion
      from the [Confluent maven repo](http://packages.confluent.io/maven/) by replacing the repositories
```

Reply:
Hey @JimGalasyn , does hiding the URL behind the text "Confluent maven repo" make the note harder to use, in reference to "replacing the repositories in the example POM above with a repository with this URL instead"? 

I'm happy with your suggestions if you think it's better -- just wanted to point out my original motivation for leaving the URL visible.

Reply:
Ah I get it! Generally we want URLs to be hot, so we could have the best of both worlds by changing the link text to the URL:  [http://packages.confluent.io/maven/](http://packages.confluent.io/maven/). How does that sound?

Reply:
The best of both worlds indeed! I'll make the updates. Thanks!